### ISSUE TITLE
Enhancement: Add example to README on how to access the lambdas

### ISSUE THREAD

**Issue by johnweland** [rileydakota/valheim-ecs-fargate-cdk#23]
Forgive me I am an AWS noob, I assume to hit the lambdas for starting and stopping the server I would use API Gateway and Postman? However this looks likes it is expecting an auto token. Can you flesh out a bit of a guide in the FAQ's in the README perhaps?

I see the project hasn't been updated in a number of months. I think this is an amazing project and hope it doesn't fall to the wayside. Let me know how I can help if at all.

**Comment by rjaduthie** [comment]
You're right that this is not clear from the documentation. Some improvement could be made.  I might take some time to do this in the next few days.  Maybe it could do something like:

---

Once you have deployed `ValheimServer/LambdaEcsFargateUpdownstatusStack`, you can control the server by using:

_NB., the API domain is returned when running `npx cdk deploy ValheimServer/LambdaEcsFargateUpdownstatusStack` but is also available after the stack is deployed in the API Gateway section of the AWS web console_

### Get server status
`https://[YOUR_API_DOMAIN]/prod/serverstatus`

### Stop server
`https://[YOUR_API_DOMAIN]/prod/startstop?key=[PASSWORD_SET_IN_DOTENV]&desireCount=0`

### Start server
`https://[YOUR_API_DOMAIN]/prod/startstop?key=[PASSWORD_SET_IN_DOTENV]&desireCount=1`
